bip-0373.mediawiki

```diff
@@ -46,53 +46,53 @@ The new per-input types are defined as follows:
 | rowspan="2"|MuSig2 Participant Public Keys
 | rowspan="2"|<tt>PSBT_IN_MUSIG2_PARTICIPANT_PUBKEYS = 0x1a</tt>
 | <tt><33 byte plain aggregate pubkey></tt>
-| <tt><33 byte compressed pubkey>*</tt>
+| <tt><33 byte plain participant pubkey>*</tt>
 | rowspan="2"|
 | rowspan="2"|
 | rowspan="2"| 0, 2
 |-
-| The MuSig2 aggregate plain public key<ref>'''Why the plain aggregate public key instead of x-only?'''
+| The MuSig2 plain aggregate public key<ref>'''Why the plain aggregate public key instead of x-only?'''
 BIP 32 requires public keys to include their evenness byte. Aggregate public keys are expected to be
 derived from, following [[bip-0328.mediawiki|BIP 328]], and therefore will
 need to include the evenness. Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
-to identify master keys, and these fingerprints require full compressed public keys. By including
+to identify master keys, and these fingerprints require full plain public keys. By including
```

Comment:
require the y-coordinate of the public key, so x-only serialisation can't be used.